frostsnap_core/tests/malicious.rs

```diff
@@ -24,21 +25,23 @@ fn keygen_maliciously_replace_public_poly() {
     let mut run = Run::generate(1, &mut test_rng);
     let device_set = run.device_set();
     let mut shadow_device = run.devices.values().next().unwrap().clone();
-    let coordinator_keygen_keypair = FrostCoordinator::short_lived_keygen_keypair(&mut test_rng);
+
+    use rand_chacha::rand_core::{RngCore, SeedableRng};
+    let mut seed = [0u8; 32];
+    test_rng.fill_bytes(&mut seed);
+    let mut coordinator_rng = ChaCha20Rng::from_seed(seed);
```

Comment:
This is necessary because otherwise proptest can end in a situation where a device and the coordinator have the same public key which causes weird stuff to happen (nothing insecure).